ouroboros-network/src/Ouroboros/Network/PeerSelection/State/LocalRootPeers.hs

```diff
@@ -72,6 +73,12 @@ newtype WarmValency = WarmValency { getWarmValency :: Int }
   deriving (Show, Eq, Ord)
   deriving Num via Int
 
+-- | Data available from topology file.
+--
+type Config peeraddr =
+     [(HotValency, WarmValency, Map peeraddr ( PeerAdvertise, PeerTrustable))]
```

Comment:
I am not a fan of type alias in general, I like this idea though so maybe having a newtype is better so we make sure we always use it and not make the code be more confusing by using `Config` in one place and not others.

Perhaps other idea is to have :

```haskell
data Config = Config { hotValency  :: HotValency
                     , warmValency :: WarmValency
                     , peers       :: Map peeraddr ( PeerAdvertise, PeerTrustable)) 
                     }
```

and then use `[LocalRootPeers.Config]` instead.

Reply:
I know, but I thought the idea was to type less 😛